### PR TITLE
remove polyfills for unsupported php versions

### PR DESCRIPTION
**Pull request by Tobion**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

Remove obsolete polyfills in master as introduced in #16317

### CODE REVIEW

File: src/Symfony/Component/Security/Core/composer.json [hunk 1]
```diff
@@ -17,9 +17,7 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/polyfill-php55": "~1.0",
         "symfony/polyfill-php56": "~1.0",
-        "symfony/polyfill-php70": "~1.0",
```
Comment (Tobion) [inline src/Symfony/Component/Security/Core/composer.json]: https://github.com/symfony/symfony/pull/16365#discussion_r43325863

File: src/Symfony/Component/Asset/composer.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.5.9"
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Asset/composer.json]: this should even be changed in 2.7/2.8 to 5.3.9, isn't it?

File: src/Symfony/Component/Security/composer.json [hunk 3]
```diff
@@ -20,7 +20,6 @@
         "symfony/event-dispatcher": "~2.8|~3.0",
         "symfony/http-foundation": "~2.8|~3.0",
         "symfony/http-kernel": "~2.8|~3.0",
-        "symfony/polyfill-php55": "~1.0",
         "symfony/polyfill-php56": "~1.0",
         "symfony/polyfill-php70": "~1.0",
         "symfony/polyfill-util": "~1.0",
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/composer.json]: extra coma on the line below

File: src/Symfony/Component/Asset/composer.json (hunk 2 above)
Comment (Tobion) [inline src/Symfony/Component/Asset/composer.json]: I will do that

File: src/Symfony/Component/Security/composer.json (hunk 3 above)
Comment (Tobion) [inline src/Symfony/Component/Security/composer.json]: fixed

File: src/Symfony/Component/Asset/composer.json (hunk 2 above)
Comment (Tobion) [inline src/Symfony/Component/Asset/composer.json]: done and rebased

**Pull request merged**